src/pomodoro.sh

```diff
@@ -85,10 +85,13 @@ function show_active_pomodoro_timebox()
 # Show registered tags with number identification.
```

Comment:
Also, think you forgot to add verbose to `select_from` in the function `get_tag_name`.